deployment/sample-config.yml

```diff
@@ -1,8 +1,8 @@
 # Supported operations are 'add-layer', 'update-version', 'update-env-variables', 'disable', 'enable'
 operation: add-layer
 
-# Layer arn needs to be specified for 'add' or 'update-version' operations
-layer_arn: arn:aws:lambda:us-west-2:342994379019:layer:asserts-aws-lambda-layer-py:3
+# Layer arn needs to be specified for 'add-layer' or 'update-version' operations
+layer_arn: arn:aws:lambda:us-west-2:689200961921:layer:asserts-aws-lambda-layer-python:1
```

Comment:
i assume 689200961921 is a shared aws account

Reply:
It's our prod account. The layers have been made public and can be accessed from other AWS Accounts.